.eslintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/**
+/template/src**
+/template/bin**
```

Comment:
What's that?

Reply:
Remnant of the previous logic - removed now